themes/aframe/source/css/secondary.styl

```diff
@@ -161,6 +161,24 @@ a
 .section-subnav
     margin-bottom 10px
 
+.post__projects
+    display flex
+    flex-wrap wrap
+    margin-bottom 15px
+    a
+        display block
+        height 160px
+        margin 1px
+        width unquote('calc(50% - 2px)')
```

Comment:
it'd be nice to have the layout be responsive to page width:

![image](https://cloud.githubusercontent.com/assets/203725/14442888/fdf07262-fff0-11e5-8d98-8c170d725aa6.png)

we can tweak this later. legit for now.